pkg/agent/core/ngt/handler/grpc/object_test.go

```diff
@@ -1185,6 +1188,164 @@ func Test_server_GetObject(t *testing.T) {
 	}
 }
 
+type ListObjectStreamMock struct {
+	mock.ServerStreamTestifyMock
+}
+
+func (sssm *ListObjectStreamMock) Send(res *payload.Object_List_Response) error {
+	args := sssm.Called(res)
+	return args.Error(0)
+}
```

Comment:
Is there any reason to define the mock implementation here instead of internal/test/mock package?

Reply:
[f64b1c4](https://github.com/vdaas/vald/pull/2145/commits/f64b1c4ad2011f614e2cf7b4f1008f3950da13dd)